src/main/java/pl/byczazagroda/trackexpensesappbackend/exception/ErrorCode.java

```diff
@@ -9,6 +9,8 @@ public enum ErrorCode {
 
     FT001("FT001", "FINANCIAL_TRANSACTION_NOT_FOUND", 404),
 
+    FTC001("FTC001", "FINANCIAL_TRANSACTION_CATEGORY_NOT_FOUND", 404),
```

Comment:
The value of the error code is
`FINANCIAL TRANSACTION CATEGORY NOT FOUND`

But in `FinancialTransactionCategoryServiceImpl` class error message is:
```java
String.format("Financial transaction category with given id: %d does not exist", id));
```

Should the convention be the same?
- not found
or
- not exist?

Reply:
I was suggested by that line:

`String.format("FinancialTransaction with given id: %d does not exist", id));`

What's the final convention?

Reply:
> I was suggested by that line:
> 
> `String.format("FinancialTransaction with given id: %d does not exist", id));`
> 
> What's the final convention?

Check how it is now in the code:

1. Search the project for occurrences of words:
a) "String.format"
b) "not exist"
c) "not found"
Then you'll see if the convention repeats itself.

2. Check if there is, and if so, what convention is inside the enum ErrorCode.

Then you decide by yourself. And if there is no convention anywhere, then it's worth raising the topic on the #os-backend channel. To do so, create a thread under this channel.
The advantage of such a solution in the worst case will be submitting the task for later naming refactoring.